### DISCUSSION TITLE
Error: undefined symbol: secp256k1_ecdsa_recoverable_signature_parse_compact

### DISCUSSION THREAD

**Discussion by winsvega** [bitcoin-core/secp256k1#1448] [Q&A]
I am getting this error after I linked the .so libfile:
What can it be? 

`error: undefined symbol: secp256k1_ecdsa_recoverable_signature_parse_compact`

I compiled using this 
```
export ANDROID_NDK_ROOT=/home/wins/Android/android-ndk-r25c/ 
cmake .. -DCMAKE_TOOLCHAIN_FILE="${ANDROID_NDK_ROOT}/build/cmake/android.toolchain.cmake" -DANDROID_ABI=arm64-v8a -DANDROID_PLATFORM=28
```

**Comment by real-or-random** [comment]
You need to enable the recovery module, which is disabled by default. Try passing `-DSECP256K1_ENABLE_MODULE_RECOVERY=ON` to CMake.
